intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -214,6 +216,70 @@ def convert_date(value):
     return harmonization.DateTime.sanitize(value)
 
 
+# https://www.shadowserver.org/wiki/pmwiki.php/Services/Open-DB2
+scan_db2 = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port')
+    ],
+    'optional_fields': [
+        ('protocol.transport', 'protocol'),
+        ('source.reverse_dns', 'hostname'),
+        ('source.asn', 'asn'),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('destination.reverse_dns', 'db2_hostname'),
+        ('extra.', 'naics', invalidate_zero),
+        ('extra.', 'sic', invalidate_zero),
+        ('extra.', 'size', convert_int),
+        ('extra.', 'servername', validate_to_none),
+        ('extra.', 'tag', validate_to_none)
```

Comment:
We usually don't store 'tag'

Reply:
If you don't set it explicitly, it will be stored in `extra` anyway

Reply:
Sorry, forgot about that. :-/

Reply:
Ok, I will remove it to be consistent